README.md

```diff
@@ -1 +1,20 @@
-# identity-sanity-tests
+# Identity API Sanity Tests
+
+Sanity tests of Identity API REST endpoints. 
+
+Tests execute against Web and Mobile Apps hosts in different stages (CODE, PROD). 
+Stage targeting is done via an environmental variable:
+
+```
+ID_API_TARGET_STAGE := CODE | PROD
+```
+Domain names of targets are set in `application.conf`
+
+List of targets:
+
+Consumer | Stage | Domain
+---------|-------|--------------------------------------
+Web      | CODE  | https://idapi.code.dev-theguardian.com
+Web      | PROD  | https://idapi.theguardian.com
+Mobile   | CODE  | https://id.code.dev-guardianapis.com
+Mobile   | PROD  | https://id.guardianapis.com
```

Comment:
The consumer part isn't strictly true, whilst each of those is one consumer, its not the only one. I don't think we should have consumers within this at all. All we need is the domain as a key to a map, the value of which is a list of domains to be used.

Map(
"CODE" -> List("https://idapi.code.dev-theguardian.com", "https://id.code.dev-guardianapis.com"),
"PROD" -> List("https://idapi.theguardian.com", "https://id.guardianapis.com")
)